storage/rocksdb/rdb_native_dd.cc

```diff
@@ -0,0 +1,54 @@
+/*
+   Copyright (c) 2012,2013 Monty Program Ab
```

Comment:
```suggestion
   Copyright (c) 2023 Meta, Inc
```
or whatever the requirement is :)